.github/workflows/build-and-publish.yml

```
@@ -0,0 +1,167 @@
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '8.0.x'
+        
+    - name: Extract version from tag
+      id: version
+      run: |
+        # Extract version from tag (v1.2.3 -> 1.2.3)
+        VERSION=${GITHUB_REF#refs/tags/v}
+        echo "version=$VERSION" >> $GITHUB_OUTPUT
+        echo "Version extracted from tag: $VERSION"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)